internal/operators/manager.go

```diff
@@ -448,3 +473,17 @@ func EnsureLVMAndCNVDoNotClash(cluster *common.Cluster, openshiftVersion string,
 	}
 	return nil
 }
+
+func (mgr *Manager) EnsureOperatorPrerequisite(cluster *common.Cluster, openshiftVersion string, operators []*models.MonitoredOperator) error {
+	err := EnsureLVMAndCNVDoNotClash(cluster, openshiftVersion, operators)
+	if err != nil {
+		return err
+	}
+
+	err = mgr.EnsureOperatorArchCapability(cluster, operators)
```

Comment:
maybe we can skip this check if OCP version < 4.11, WDYT?

Reply:
To my opinion this have the same affect and the ~same running time. 
Only if we want to make it be more explicit (but it will make the code more complex over time)